torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -657,23 +657,20 @@ def get_attr(
         ],
         fx_graph_module: torch.fx.GraphModule,
     ):
-        current_attr = fx_graph_module
```

Comment:
I'm removing this parameter lookup from submodules because this is dead code. We do not get fx.GraphModule like this in our current code path.